bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -475,18 +433,19 @@ class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
 	                                     std::optional<int> tenantId,
 	                                     int64_t readVersion) {
 		ASSERT(!results.empty());
-		ASSERT(results.front().keyRange.beginKey <= keyRange.beginKey);
-		ASSERT(keyRange.endKey <= results.back().keyRange.endKey);
-		for (int i = 0; i < results.size() - 1; i++) {
-			ASSERT(results[i].keyRange.endKey == results[i + 1].keyRange.beginKey);
-		}
 
 		if (tenantId) {
 			// FIXME: support tenants!!
```

Comment:
Is it difficult to validate results in presence of tenants?

Reply:
This PR is just prior to adding tenant support, I have another follow-up PR that adds encryption and tenant support